### ISSUE TITLE
Expanded Off-season News Stories -- Multi Year Stories

### ISSUE THREAD

**Issue by destilla** [jonesguy14/footballcoach#47]
TL;DR : Multi year stories would add some continuity and ongoing story lines to a user's dynasty.

Adding additional stories won't be more than a bit of work and some creative thinking, but a user on the  subreddit gave me the great idea of doing multiple year stories similar to the Chicago Cub's curse or what happened with UAB being dismantled slowly then brought back to life. Essentially:

TEAM loses some prestige, a story is kicked off about something legitimately cursing the team or threatening it's existence/place of power in CFB. A variable noting what year of the subplot we're is advanced and then basically progresses through it. First the baseball team goes winless. Then the statue of beloved hall of fame coach RAND.NAME is struck by lightning. Etc etc as they slowly lose bits and pieces of prestige for a couple of years until they've hit the amount of they would have lost in year 1 if they had just been cursed (or maybe a bit more?). 

Suddenly things turn around (UAB came back to life, the Red Sox eventually won a world series, Philly sports....well nvm) and maybe they're granted a small prestige bonus (1 or 2) and they can start the healing process.

Once the scaffolding for this is in place, other flavors of story could be added. Maybe hiring that new coach was more than just Xs and Os, maybe it's been a culture change at the school as a whole. Maybe other stories that I'm not creative enough to think of right now.

If you think this is too ambitious or don't like it, feel free to close the issue or maybe tag it as a long term "nice to have" type addition.

**Comment by jonesguy14** [comment]
Multi year storylines could be cool, but we'd have to think more about how the prestige bonus/penalty is handled. In fact, right now, I've noticed that "cursed" teams don't seem to be affected greatly by the penalty. 

Say, for example ALA loses 30 prestige. Now they have 65 prestige. The next year, they recruited freshman badly due to their low prestige, but still have a really good roster from years past where their prestige was 90+. So they will still win a lot of games even with the low prestige, which means they will gain a great deal of prestige for exceeding expectations. Within a couple years they are back up to 80 or more prestige, and just had one bad recruiting class. This can happen in reverse with bad teams that are blessed, where they gain a ton of prestige, fall short of expectations, lose prestige, etc.

So before we make multi year storylines I think we should think more about how to really penalize or really give a bonus to teams in a meaningful way to promote variety in which teams can become contenders. (Maybe I'll put it in a separate issue?) I do like the idea though!
